### ISSUE TITLE
BNF grammar for WARC-Target-URI and WARC-Profile is inconsistent with examples

### ISSUE THREAD

**Issue by ato** [iipc/warc-specifications#23]
Sections 5.2 and 5.16 define the grammar for these fields as:

```
WARC-Record-ID = "WARC-Record-ID" ":" uri
WARC-Profile   = "WARC-Profile" ":" uri
WARC-Target-URI = "WARC-Target-URI" ":" uri
```

Section 4 defines `uri` as:

```
uri            = "<" <'URI' per RFC3986> ">"
```

However all examples in sections 12 and 13 do not include the "<" and ">" characters on the WARC-Target-URI and WARC-Profile fields. All other `uri` fields (WARC-Record-ID, WARC-Refers-To etc) in the examples include "<" and ">".

```
WARC/1.0
WARC-Type: revisit
WARC-Target-URI: http://www.archive.org/images/logoc.jpg
WARC-Date: 2007-03-06T00:43:35Z
WARC-Profile: http://netpreserve.org/warc/1.0/server-not-modified
WARC-Record-ID: <urn:uuid:16da6da0-bcdc-49c3-927e-57494593bbbb>
WARC-Refers-To: <urn:uuid:92283950-ef2f-4d72-b224-f54c6ec90bb0>
Content-Type: message/http
Content-Length: 226
```

Many (all?) implementations have adopted the form shown in the examples rather than strictly following the grammar.

**Comment by kris-sigur** [comment]
Yes, current tools follow the examples as far as I can tell. The `< >` form is only used when dealing with UUIDs.

I suspect most tools would choke on something that is formatted as per the BNF.

**Comment by nclarkekb** [comment]
I always though the point was that target-uri and profile where most likely URLs that could be browsed to. And the rest were more likely UUIDs and hence surrounded by "<" / ">".
Both being URIs was more a technicallity.

**Comment by saraaubry** [comment]
The following changes have been integrated in the revised ISO draft during the ISO working group meeting on November 16-17, 2015:

in section 4 file and record model, change the definition of uri:
uri            = <'URI' per RFC3986>

NOTE: in WARC 1.0 standard (ISO 28500:2009), uri was defined as "<" <'URI' per RFC3986> ">". This rule has been changed to meet requests from implementers.

**Comment by saraaubry** [comment]
Included in WARC 1.1
